framework/addons/data/addons.json

```diff
@@ -0,0 +1,86 @@
+{
+    "addons": {
+        "GitHub": {
+            "Permissions": "partial | Making an OSF project public or private is independent of making a GitHub repo public or private. The OSF does not alter the permissions of linked GitHub repos.",
```

Comment:
Instead of using an arbitrary delimiter, why not use an object?

```
"Permissions" : {
    {"status": "partial", "detail": "Making an OSF project ...."
}
```